### ISSUE TITLE
wrong opam path in doc

### ISSUE THREAD

**Issue by sanette** [OCamlPro/ocp-indent#241]
Hi 

in the webpage https://www.typerex.org/ocp-indent.html
it is indicated:

> The above installation step copies elisp and vim scripts to <prefix>/share/typerex/ocp-indent/. You then need to load them in the editor of your choice to automatically use ocp-indent.

However, after install ocp-indent from opam, the file:
$(opam config var prefix)"/share/typerex/ocp-indent/ocp-indent.el"
does not exist. It should be instead:
$(opam config var prefix)"/share/emacs/site-lisp/ocp-indent.el"

**Comment by AltGr** [comment]
Indeed, this has been changed but the documentation wasn't updated. Thanks for notifying!
